.github/workflows/test-upstream.yaml

```diff
@@ -53,7 +53,9 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Install poetry
-        run: pipx install poetry
+        # Pin to 1.8.5. because Poetry 2.0.0 installs plugins in a different way
+        # (dynamic versioning in next step)
+        run: pipx install poetry==1.8.5
```

Comment:
It looks like there is a way to pin a specific version of poetry in pyproject.toml https://python-poetry.org/blog/announcing-poetry-2.0.0/

We may want to do a quick 1-line PR to upstream linkml to add this pin there rather than here so they don't go out of sync & fix later, but since this is likely to be a quick and easy change to both packages not necessary at all and this would be fine too

Reply:
I believe the option to[ constrain poetry](https://python-poetry.org/docs/pyproject#requires-poetry) was added only in 2.0 - I could not find it in 1.8.5.